runtime/north/cc_node.c

```diff
@@ -663,7 +664,7 @@ result_t node_run(node_t *node)
 							node->state = NODE_PENDING;
 					}
 #else
-					platform_evt_wait(node, 0);
```

Comment:
Why a timeout of 1 second? Passing 0 was used to block indefinitely until there was any data on a transport client.